src/parsing/refactored_2_1/backups/doc_ingest_pipeline_key_changes.py

```
@@ -27,9 +24,7 @@ async def ingest_sources(
     # Initialize components
     cache = CacheManager() if config.enable_cache else None
     validator = DocumentValidator(config)
-    progress = ProgressMonitor(
-        callback=_progress_callback if config.progress_callback else None
-    )
+    progress = ProgressMonitor(callback=_progress_callback if config.progress_callback else None)
```

Ruff (0.11.9)

Undefined name `_progress_callback`

(F821)